cmd/content-sources/main.go

```diff
@@ -7,19 +7,27 @@ import (
 	"os/signal"
 	"sync"
 	"syscall"
+	"time"
 
 	"github.com/content-services/content-sources-backend/pkg/config"
 	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/content-services/content-sources-backend/pkg/event"
 	eventHandler "github.com/content-services/content-sources-backend/pkg/event/handler"
 	"github.com/content-services/content-sources-backend/pkg/handler"
+	m "github.com/content-services/content-sources-backend/pkg/instrumentation"
+	"github.com/labstack/echo/v4"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/zerolog/log"
 )
 
 func main() {
+	reg := prometheus.NewRegistry()
+	metrics := m.NewMetrics(reg)
```

Comment:
I see what you mean, thanks. When I did this, I used a global variable, which I liked because it kept everything really simple. But if it's important to avoid that, I see why dependency injection would be useful. I also wasn't thinking about unit tests :)